app/src/main/res/values/strings.xml

```diff
@@ -623,7 +623,11 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="quest_railway_crossing_barrier_none">It\'s not</string>
     <string name="action_deselect_all">Deselect all</string>
     <string name="quest_buildingType_retail_description">including buildings for restaurants, cafés, …</string>
-
+    <string name="quest_maxweight_title">"What is the weight limit here?"</string>
```

Comment:
Hmm, here is a quotation character too much, is not it? This should not be quoted…